packages/table/examples/tableSortableExample.tsx

```diff
@@ -46,6 +47,7 @@ abstract class AbstractSortableColumn {
         return (<Column
             key={this.index}
             name={this.name}
+            loadingOptions={new Set([ ColumnLoadingOption.COLUMN_HEADER, ColumnLoadingOption.CELL ])}
```

Comment:
These shouldn't be part of this example. Please add an example to `packages/table/preview/index.tsx` for dev testing and a new example to the table docs section.